File: Assignments/Assignment-14/p1.c
#include<stdio.h>
void main(){

        int row;
        printf("Enter the no of rows : \n");
        scanf("%d",&row);
        for(int i=1;i<=row;i++){

                for(int j=1;j<=row;j++){

                        printf("%d\t",(i*j));
                }printf("\n\n");
        }
}

File: Flutter/Assignment/Square pattern set-2/p1.dart
import "dart:io";
void main(){
	print("Enter the no. of rows:");
	int? row=int.parse(stdin.readLineSync()!);
	for(int i=1;i<=row;i++){
		for(int j=1;j<=row;j++){
			if(j<=i){
				stdout.write("${i}\t");
			}else{
				stdout.write("*\t");
			}
		}
		print("");
	}
}
